### PR TITLE
Tabs: Add stylingMode option

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js [hunk 1]
```diff
@@ -318,6 +326,19 @@ QUnit.module('General', () => {
             assert.ok($element.hasClass(TABS_ICON_POSITION_CLASS[iconPosition]));
         });
     });
+
+    QUnit.test('the tabs element must have a correct styling mode class', function(assert) {
+        const $element = $('#tabs').dxTabs({ items: [1, 2, 3] });
+        const instance = $element.dxTabs('instance');
+
+        assert.strictEqual(instance.option('stylingMode'), STYLING_MODE.primary);
+        assert.strictEqual($element.hasClass(TABS_STYLING_MODE_CLASS.primary), true);
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js:335]: We don't check that other styling mode classes not added to element.

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js [hunk 2]
```diff
@@ -318,6 +326,19 @@ QUnit.module('General', () => {
...
+
+        instance.option({ stylingMode: 'secondary' });
+
+        assert.strictEqual(instance.option('stylingMode'), STYLING_MODE.secondary);
+        assert.strictEqual($element.hasClass(TABS_STYLING_MODE_CLASS.secondary), true);
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js:341]: We don't check that 'dx-tabs-styling-mode-primary' class was removed from element.

File: packages/devextreme-angular/src/ui/tabs/index.ts [hunk 3]
```diff
@@ -390,6 +390,19 @@ export class DxTabsComponent extends DxComponent implements OnDestroy, OnChanges
     }
 
 
+    /**
```
Comment (EugeniyKiyashko) [inline packages/devextreme-angular/src/ui/tabs/index.ts:393]: Can I find info in DX WIKI `what I should add in wrappers` when I add new options to component? If yes, please, share this knowledge between colleagues

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js (hunk 1 above)
Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js:335]: Fixed here [48547431cb91e50f04bb940df4b4dd7095916eaa]

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js (hunk 2 above)
Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js:341]: Fixed here [48547431cb91e50f04bb940df4b4dd7095916eaa]

File: packages/devextreme-angular/src/ui/tabs/index.ts (hunk 3 above)
Comment (marker-dao) [reply packages/devextreme-angular/src/ui/tabs/index.ts:393]: You should use npm command: npm run regenerate-all from root.

**Review by EugeniyKiyashko — DISMISSED** [review]

**Review by vladaskorohodova — DISMISSED** [review]

**Review by alexanderPolosatov — COMMENTED** [review]
LGTM

**Review by EugeniyKiyashko — DISMISSED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**